docs/get-started.md

```diff
@@ -1,170 +1,64 @@
-Getting Started
-===============
-
-In order to consume these components you will require a system that
-utilised a bundler (such as [Webpack]) that can process imported assets
-such as images, fonts and [Sass] files. [Create React App] may be able to
-do this but does not provide Server-Side Rendering (SSR). You will also
-need to ensure that you provide an instance of [react-router].
-
-As such, we suggest that you use our specially designed tech stack for
-this purpose. You can start a brand new project or prototype using the
-following steps.
+Prototyping
+===========
 
+This guide explains how to create prototypes using the Home Office Design System.
 
 ### 1. Set up your repository
 
-You should first set up a blank code repository in the provider of your
-choice. For the purposes of this document I will assume [GitHub] as we
-provide extra CI support through [GitHub Actions].
+Set up a blank code repository in [GitHub] as
+extra CI support is provided through [GitHub Actions].
 
 
 ### 2. Run the following commands locally
 
-(The parts in ALL CAPS should be replaced with something specific to
-you.)
+Using terminal, run the following commands below (the parts in ALL CAPS should be replaced with something specific to
+you):
 
 ```shell
-mkdir YOUR-PROJECT
-cd YOUR-PROJECT
+mkdir YOUR-PROTOTYPE
+cd YOUR-PROTOTYPE
 git init
-git remote add origin git@github.com:YOUR_USER/YOUR-PROJECT.git
+git remote add origin git@github.com:YOUR_USER/YOUR-PROTOTYPE.git
 npm init @hods
 ```
 
 
 ### 3. Answer the interactive prompts
 
-Our generator / initialiser will ask you some questions about your new
-project.
+The generator / initialiser will ask you some questions about whether you want to create a new prototype or
+project:
```

Comment:
We could remove this and instead tell them to run:
```shell
npm init @hods prototype
```

Reply:
Updated now. Could you review?

Reply:
You've updated the command but you still need to remove the stuff where we explain the choice. (As they have already made their choice with the new command.)

Reply:
> The generator / initialiser will ask you some questions but the defaults should be fine. You can accept the default by simply pressing enter.

Reply:
Removed the content in this [PR](https://github.com/UKHomeOffice/hods-poc/pull/57)